airbyte-integrations/connectors/source-primetric/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,1495 @@
+                "null",
+                "integer"
+              ]
+            },
+            "work_by_capacity_per_friday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_by_capacity_per_saturday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_by_capacity_per_sunday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_per_monday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_per_tuesday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_per_wednesday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_per_thursday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_per_friday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_per_saturday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_per_sunday" : {
+              "type" : [
+                "null",
+                "integer"
+              ]
+            },
+            "work_on_monday" : {
+              "type" : [
+                "null",
+                "boolean"
+              ]
+            },
+            "work_on_tuesday" : {
+              "type" : [
+                "null",
+                "boolean"
+              ]
+            },
+            "work_on_wednesday" : {
+              "type" : [
+                "null",
+                "boolean"
+              ]
+            },
+            "work_on_thursday" : {
+              "type" : [
+                "null",
+                "boolean"
+              ]
+            },
+            "work_on_friday" : {
+              "type" : [
+                "null",
+                "boolean"
+              ]
+            },
+            "work_on_saturday" : {
+              "type" : [
+                "null",
+                "boolean"
+              ]
+            },
+            "work_on_sunday" : {
+              "type" : [
+                "null",
+                "boolean"
+              ]
+            }
+          },
+          "supported_sync_modes" : [
+            "full_refresh"
+          ],
+          "source_defined_cursor" : true,
+          "default_cursor_field" : [
+            "created_at"
+          ],
+          "source_defined_primary_key" : [
+            [
+              "uuid"
+            ]
+          ]
+        }
+      },
+      "sync_mode" : "incremental",
```

Comment:
All sync modes should be full_refresh ?

Reply:
Yes, currently Primetric does not have fields that are necessary for incremental sync such as created_at / updated_at for streams such as **Assignments**, **Capacities**, **Employees**, **People**, **Projects**, **Projects_vacancies**, **Skills**, **Timeoffs** and **Worklogs** . I asked if they are planning to have such fields in the future so that streams where there is more data can be loaded incrementally, not in full batch but for now the priority for that is low. I am not from Primetric and for our use case It is good enough to have a full refresh available. The amount of data in the streams is rather low so the process does not take a lot of time :smiling_face_with_tear:

Reply:
Oh, thanks for the answer. The sync_mode here should be `FULL_REFRESH`.

Reply:
Changed sync mode for all streams to full_refresh. Thanks!